staging/src/k8s.io/code-generator/cmd/client-gen/args/args.go

```diff
@@ -17,11 +17,22 @@ limitations under the License.
 package args
 
 import (
+	"io/ioutil"
+
 	"github.com/spf13/pflag"
+	yaml "gopkg.in/yaml.v2"
 
 	"k8s.io/code-generator/cmd/client-gen/types"
 )
 
+// NamerExceptions contains public, private and plural name exceptions for
+// custom types.
+type NamerExceptions struct {
+	PublicNamerExceptions  map[string]string `yaml:"publicNamerExceptions"`
```

Comment:
public, private, plural should be enough as the file is for namer exceptions only.